.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,26 +1,35 @@
 ---
-name: Feature request
-about: Suggest an idea for this project
-title: 'Feature: '
-labels: Suggestion
+name: 🚀 Feature Request
+about: Suggest an idea for AzerothCore
+title: '🚀 [Feature]'
+labels: 'feature'
 assignees: ''
 
 ---
+## 🚀 Feature
+<!-- Describe your feature request or suggestion in detail. A clear and concise description of what you want to happen. -->
 
-<!-- IF YOU DO NOT FILL THIS TEMPLATE OUT, WE WILL CLOSE YOUR ISSUE! -->
 
-<!-- This template is made for feature requests or suggestions on the project only. -->
 
-**Describe your feature request or suggestion in detail**
-<!-- A clear and concise description of what you want to happen. -->
 
-**Describe a possible solution to your Feature or Suggestion in detail**
-<!-- A clear and concise description of any alternative solutions or features you've considered. -->
+### Visualization
+<!-- Attach Screenshots and Drawings. Use https://draw.io -->
```

Comment:
This really doesnt make sense to make drawings, better to keep tell people to explain, and if they choose they could use images but to strictly limit to images, no.

Reply:
This section  was unified under the feature itself:

```
## 🚀 Feature
<!-- Describe your feature request or suggestion in detail. A clear and concise description of what you want to happen. -->


[...]
```

Naturally when you submit a feature request you propose a solution and I believe its not necessary to split into two sections as it was before:
- Describe your feature request or suggestion in detail
- Describe a possible solution to your Feature or Suggestion in detail

Furthermore I believe that encouraging people do draw something about their feature will make them think about how to explain it in a way that more people can understand it. Having a visualization can help to make people understand
> A picture is worth a thousand words
There is this saying for a reason.

Reply:
I disagree, if I am going to make s festure im not going to draw ehat I mean. For example if I want the festure to be able to teleport to instance ID there is no way to drsw that.

I get your point but believe it is not useful in wow emulation

Reply:
I think we can just have it as an optional field. For example:

`### Visualization (Optional)`

Reply:
I don't think that's smart, but done